src/wp-includes/blocks/navigation.php

```diff
@@ -1458,13 +1458,18 @@ function block_core_navigation_set_ignored_hooked_blocks_metadata( $inner_blocks
 /**
  * Updates the post meta with the list of ignored hooked blocks when the navigation is created or updated via the REST API.
  *
- * @param WP_Post $post Post object.
+ * @param stdClass $post Post object.
```

Comment:
Pretty sure I flagged this in the Gutenberg PR already

We must add:

* `@return stdClass`
* `@since 6.5.0`
* `@access private 

In the correct order of course, according to the inline docs standards.

Reply:
See https://github.com/WordPress/gutenberg/pull/59774